tree/ntuple/v7/src/RNTupleModel.cxx

```diff
@@ -20,10 +20,18 @@
 #include <TError.h>
 
 #include <cstdlib>
+#include <exception>
 #include <memory>
 #include <utility>
 
 
+void ROOT::Experimental::RNTupleModel::EnsureUniqueFieldName(const std::string& fieldName)
+{
+   if (fFieldNames.insert(fieldName).second == false) {
+      throw std::runtime_error("RNTupleModel::MakeField: field name '" + fieldName + "' already exists");
```

Comment:
Let's throw a ROOT specific `RException`

Reply:
Thanks, done in eea9c7c74fdeb2119d9b2949cd5b9fe6e6d13edc.